protected/humhub/modules/user/views/registration/index.php

```diff
@@ -1,20 +1,29 @@
 <?php
 
+use humhub\widgets\SiteLogo;
+use yii\bootstrap\ActiveForm;
```

Comment:
Not really needed but could use the extended class version which HumHub provides.

Reply:
See following class;

https://github.com/humhub/humhub/blob/master/protected/humhub/modules/ui/form/widgets/ActiveForm.php#L21